installing.rst

```diff
@@ -293,42 +298,59 @@ If you have Gauge and your language runner installed, then see how you can :ref:
 Uninstallation
 --------------
 
-Gauge
-^^^^^^^^
-
 .. warning::
    If you plan to remove Gauge and the installed plugins, follow the steps in :ref:`plugins-uninstallation` first.
 
-
+Gauge
+^^^^^
 To uninstall Gauge, run the following commands:
```

Comment:
worth noting that `.gauge` folder is not removed. User's need to remove manually.